src/main/java/uk/gov/hmcts/reform/pdf/service/client/PDFServiceClient.java

```diff
@@ -83,50 +83,22 @@ public Health serviceHealthy() {
     }
 
     private URI htmlEndpoint() {
-        return pdfServiceBaseUrl.resolve(
-            String.format("/api/%s/pdf-generator%s", version, GENERATE_FROM_HTML_ENDPOINT_PATH)
-        );
+        return pdfServiceBaseUrl.resolve("/pdfs");
     }
 
-    private HttpEntity<MultiValueMap<String, Object>> requestEntityFor(
+    private HttpEntity<String> requestEntityFor(
```

Comment:
Worth renaming this method I think.
At first I thought it requests an entity for something, while in reality it creates a 'request entity'. :)

Reply:
Factory method should include `create` in its name. Renaming